tests/conftest.py

```diff
@@ -113,15 +115,21 @@ def fail_test_on_unclosed_tasks(request: FixtureRequest):
     by the end of the test.
     """
 
-    fail_count = request.session.testsfailed
-    loop = asyncio.get_event_loop()
-    loop.set_debug(True)
+    try:
+        fail_count = request.session.testsfailed
+        loop = asyncio.get_running_loop()
```

Comment:
`get_event_loop` gives a deprecation warning that future python versions will raise an error if there is no event loop, `get_running_loop` does this already. It doesn't seem possible to check for an event loop without failing/warning if there isn't one, so we just catch the exception for the tests which don't use one.